FrontEndReact/src/View/Login/Login.js

```diff
@@ -60,6 +60,7 @@ class Login extends Component {
                 password,
             } = this.state;
 
+            email.toLowerCase();
```

Comment:
I don't think `toLowerCase` modifies the string in place. This may need to be changed to something like `email = email.toLowerCase();`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase

Reply:
@jcallison1 Good catch.  Rookie mistake on my part.  I've fixed that.

Reply:
Oops, sorry, bad suggestion. Looks like email is const and now the linter is failing. Maybe you could create a new variable instead?